frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -77,6 +77,23 @@ local symbol_prefix = {
         mem_usage = "",
         wifi_status = "",
         wifi_status_off = "",
+    },
+    compact_items = {
+        time = nil,
+        pages_left = ">",
+        -- pages_left = BD.mirroredUILayout() and "<" or ">",
```

Comment:
You can remove the BD.mirroredUILayout()  things left ini comments, and just add a comment after:
`pages_left = ">", -- will be auto-mirrored by BiDi`

But have you tried how https://www.fileformat.info/info/unicode/char/203a/index.htm look ?
Just copy/paste this: `›` (vs `>`)

Reply:
Thanks. The other character looks nice (and saves a bit of space! :) )